lib/uploader.ts

```diff
@@ -2,27 +2,34 @@
  * SPDX-FileCopyrightText: 2022 Nextcloud GmbH and Nextcloud contributors
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
-import type { AxiosError, AxiosResponse } from 'axios'
-import type { WebDAVClient } from 'webdav'
-import type { IDirectory } from './utils/fileTree'
-
-import { getCurrentUser } from '@nextcloud/auth'
-import { FileType, Folder, Permission, davGetClient, davRemoteURL, davRootPath } from '@nextcloud/files'
-import { encodePath } from '@nextcloud/paths'
-import { normalize } from 'path'
-
-import axios, { isCancel } from '@nextcloud/axios'
+import type {AxiosError, AxiosResponse} from 'axios'
+import type {WebDAVClient} from 'webdav'
+import type {IDirectory} from './utils/fileTree'
+
+import {getCurrentUser} from '@nextcloud/auth'
+import {
+	davGetClient,
+	davRemoteURL,
+	davRootPath,
+	FileType,
+	Folder,
+	Permission
+} from '@nextcloud/files'
+import {encodePath} from '@nextcloud/paths'
+import {normalize} from 'path'
+
+import axios, {isCancel} from '@nextcloud/axios'
 import PCancelable from 'p-cancelable'
 import PQueue from 'p-queue'
 
-import { getChunk, initChunkWorkspace, uploadData } from './utils/upload.js'
-import { getMaxChunksSize } from './utils/config.js'
-import { Status as UploadStatus, Upload } from './upload.js'
-import { isFileSystemFileEntry } from './utils/filesystem.js'
-import { Directory } from './utils/fileTree.js'
-import { t } from './utils/l10n.js'
```

Comment:
What should I run to reformat code? `npm run lint:fix` did nothing for me

Reply:
You can try `npx eslint --fix path/to/file`

Reply:
It works, thanx. Here [fix](https://github.com/nextcloud-libraries/nextcloud-upload/pull/1518) for `npm run lint:fix`